docs/dmriprep/hmc.md

```diff
@@ -1 +1,133 @@
-# Head Motion Correction
+# Contributing to *dMRIPrep* - the case of head-motion and eddy-currents estimation
```

Comment:
Include for rendering python cells.

```suggestion
---
jupytext:
  formats: md:myst
  text_representation:
    extension: .md
    format_name: myst
kernelspec:
  display_name: Python 3
  language: python
  name: python3
---

# Contributing to *dMRIPrep* - the case of head-motion and eddy-currents estimation

```{code-cell} python
:tags: [hide-cell]

import warnings
warnings.filterwarnings("ignore")
```
```